### ISSUE TITLE
Cursor moving to slow after hitting space in insert mode and quickly typing a character afterwards

### ISSUE THREAD

**Issue by immi-10** [neovide/neovide#2889]
**Describe the bug**
The cursor moves to slow when typing space which leads to wrong text input if quickly followed by different characters.

My setup:

```
if vim.g.neovide then
  vim.g.neovide_cursor_animate_in_insert_mode = false
end

```

**To Reproduce**
1. type in really quickly "the data" (or anything else with a space in it)

**Result**
The text is "theta" (missing the space and "da" from "data"

**Expected behavior**
The text should be  "the data"

**Desktop (please complete the following information):**
 - OS: macOS Sonoma 14.6.1
 - Neovide Version: 0.13.3
 - Neovim Version: 0.10.0

Logs:
[neovide_rCURRENT.log](https://github.com/user-attachments/files/17993260/neovide_rCURRENT.log)

**Comment by fredizzimo** [comment]
I don't think this is a Neovide bug, most likely you have some keyboard mapping or plugin causing it.

Does it behave the same way if you run `neovide -- --clean`? If, that works, check your mappings with `:imap` and/or try to disable plugins until it starts working.

**Comment by fredizzimo** [comment]
Closing due to lack of response.
